corehq/apps/app_manager/static/app_manager/js/shadow-module-settings.js

```diff
@@ -12,20 +12,27 @@ hqDefine('app_manager/js/shadow-module-settings.js', function () {
             self.selectedModule = ko.pureComputed(function () {
                 return _.findWhere(self.modules(), {uniqueId: self.selectedModuleId()});
             });
-            self.allForms = ko.pureComputed(function() {
-                return _.flatten(_.map(self.modules(), function(m) { return m.forms(); }));
+            // Find all forms that could potentially be included in the current module:
+            // Forms belonging to the source module and to the source module's children
```

Comment:
thanks for the comment. stared at the code for a while trying to figure out what was going on.